lightspeed/modules/proc_create-connection-secrets.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+
+[id="create-connection-secrets_{context}"]
+
+= Creating connection secrets
+
+Use this procedure to create secrets to connect to {PlatformName} and {ibmwatsonxcodeassistant}.
+
+.Prerequisites
+* You have installed the {PlatformNameShort} operator on the {OCP}. 
+* You have created an OAuth application in the automation controller.
+* You have obtained an API key and a model ID from {ibmwatsonxcodeassistant}. 
++
+For information about obtaining an API key and model ID from {ibmwatsonxcodeassistant}, see the link:https://cloud.ibm.com/docs/watsonx-code-assistant[{ibmwatsonxcodeassistant} documentation].
+
+.Procedure
+. Go to the {OCP}. 
+. Select *Workloads → Secrets*.
+. Click *Create → Key/value secret*.
+. From the *Projects* list, select the namespace that you created when you installed the {PlatformName} operator.
+. Create an *authorization secret* to connect to the {PlatformName}:
+.. Click *Create → Key/value secret*.
+.. In *Secret name*, enter a unique name for the secret. For example, `auth-aiconnect`.
+.. Add the following keys and their associated values individually:
++
+[cols="30%,70%",options="header"]
+|===
+|Key |Value
+
+|`auth_api_url`
+|Enter the API URL of the automation controller in the following format:
```

Comment:
use the automation controller attribute in this line: {ControllerName}

Reply:
Done